environment/src/main/java/com/sequenceiq/environment/environment/flow/start/handler/StartDatahubHandler.java

```diff
@@ -33,14 +37,15 @@ public String selector() {
     public void accept(Event<EnvironmentDto> environmentDtoEvent) {
         EnvironmentDto environmentDto = environmentDtoEvent.getData();
         try {
-            distroxService.startAttachedDistrox(environmentDto.getId(), environmentDto.getName());
+            datahubService.startAttachedDatahubClusters(environmentDto.getId(), environmentDto.getResourceCrn());
             EnvStartEvent envStartEvent = EnvStartEvent.EnvStartEventBuilder.anEnvStartEvent()
                     .withSelector(EnvStartStateSelectors.FINISH_ENV_START_EVENT.selector())
                     .withResourceId(environmentDto.getId())
                     .withResourceName(environmentDto.getName())
                     .build();
             eventSender().sendEvent(envStartEvent, environmentDtoEvent.getHeaders());
         } catch (Exception e) {
+            LOGGER.error(String.format("Error occurred during starting Datahub(s) for environment. %s", environmentDto), e);
```

Comment:
Duplicated log message. Failer handler logs the exception and message.

Reply:
it's not duplicated because it contains more information. entire dto is printed.

Reply:
Please update the common log message with the details

Reply:
Fixing it in the next push